src/containers/profiles/EditProfiles/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import M from 'materialize-css';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
+import avatar2 from '../../../assets/images/avatar2.png';
+import { editUserProfileAction } from '../store/profileActions';
+import Modal from '../../../components/Modal';
+import './EditProfiles.scss';
+
+class EditProfiles extends Component {
+  state = {
+    bio: '',
+    image: avatar2,
+  };
+
+  componentDidMount = () => {
+    const modals = document.querySelector('.modal');
+    M.Modal.init(modals, {});
+  }
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    const { bio, image } = this.state;
+    const data = new FormData();
+    data.append('bio', bio);
```

Comment:
Avoid mutating variables instead use the spread operator if you need to add new items.

Reply:
Thank you. Done